modules/test/playwright/tests/commerce/commerce-order-content-web/placedOrder.spec.ts

```diff
@@ -0,0 +1,138 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../../fixtures/apiHelpersTest';
```

Comment:
I think this is not needed because you're already importing the dataApiHelpersTest